lib/cluster.js

```diff
@@ -337,7 +337,7 @@ function masterInit() {
       process: workerProcess
     });
 
-    worker.on('message', this.emit.bind(this, 'message'));
+    worker.on('message', (message, Handle) => this.emit(message, Handle));
```

Comment:
and I think the correct way to put it here is `worker.on('message', (message, handle) => this.emit('message', message, handle));`